config.json

```diff
@@ -162,5 +162,17 @@
         "minimumBlockWidth": 20
       }
     ]
-  }
+  },
+  "internetAccounts": [
+    {
+      "type": "GDCInternetAccount",
```

Comment:
if testing against core this needs to change to

"type": "ExternalTokenInternetAccount",